src/GuzzleHttp/Command/Guzzle/GuzzleClient.php

```diff
@@ -0,0 +1,45 @@
+<?php
```

Comment:
Hmm can you provide me with an example? I did this way because of the paramconverter and because i need my own processresponse with the serializer!